opal-core-api/src/main/java/org/obiba/opal/core/support/genotypes/GenotypesSummaryBuilder.java

```diff
@@ -0,0 +1,95 @@
+package org.obiba.opal.core.support.genotypes;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import org.obiba.magma.Datasource;
+import org.obiba.magma.ValueSet;
+import org.obiba.magma.ValueTable;
+import org.obiba.opal.core.domain.GenotypesMapping;
+import org.obiba.opal.spi.vcf.VCFStore;
+
+import java.util.Collection;
+import java.util.Set;
+
+public class GenotypesSummaryBuilder {
```

Comment:
This is a VCF samples mapping, so rename accordingly (including the package name).